modules/ROOT/pages/flex-gateway-delete.adoc

```diff
@@ -0,0 +1,24 @@
+= Delete a Flex Gateway
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-disabled.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-active.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-active.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+To manually delete a gateway and its replicas in Runtime Manager:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Flex Gateway* in the left menu.
+. Click the action menu button for the gateway you want to unregister.
+. Click *Delete gateway*.
+
+[NOTE]
+Replicas are automatically unregistered after a period of inactivity of 10 minutes.
+
+Deleting a gateway from the Runtime Manager UI simply removes the connection between Anypoint Platform and the gateway and all its replicas. This action does not uninstall the gateway from the server.
```

Comment:
**simply** removes > removes... 
as suggested by the style guide.

Reply:
topic removed from GA